transactor/src/main/scala/io/mediachain/transactor/JsonSerialization.scala

```diff
@@ -0,0 +1,177 @@
+package io.mediachain.transactor
+
+import io.atomix.catalyst.buffer.{BufferInput, BufferOutput}
+import io.atomix.catalyst.serializer.{Serializer, TypeSerializer}
+import io.mediachain.transactor.Dummies.DummyReference
+import io.mediachain.transactor.Types._
+import org.json4s._
+import org.json4s.jackson.JsonMethods.{compact, render, parse}
+
+/**
+  * Represents a 'multihash' link to an IPFS resource
+  *
+  * @param multihash - the base58 encoded string representation of an IPFS
+  *             multihash
+  */
+case class IPFSReference(multihash: String) extends Reference
```

Comment:
agreed on this, we should have a multihash that's a bytestring

I would say let's alias `Multihash` as `String` for now and then swap the underlying later?